### ISSUE TITLE
AppImage segmentation fault

### ISSUE THREAD

**Issue by xruth** [zulip/zulip-desktop#539]
---

- **Operating System**:
  - [ ] Windows
  - [X] Linux/Antergos 18.7
  - [ ] macOS
- **Clear steps to reproduce the issue**:
Download Zulip-2.3.5-x86_64.AppImage. Run from terminal.
- **Relevant error messages and/or screenshots**:
$ ./Zulip-2.3.5-x86_64.AppImage 
Segmentation fault (core dumped)

AppImage of Zulip client does not start. Running from terminal shows segmentation fault.

**Comment by akashnimare** [comment]
@xruth did you make the file executable first using -  
> chmod a+x Zulip-x.x.x-x86_64.AppImage

**Comment by timabbott** [comment]
@akashnimare I don't think segmentatin fault is a plausible error for not making it executable.  @xruth is your system an x86_64 system?  (`uname -a` would be helpful)

**Comment by akashnimare** [comment]
I can't reproduce this issue on Ubuntu 16.04 (64-bit). @abhigyank can you repro?

**Comment by abhigyank** [comment]
Can't repro @akashnimare . Probably due to the OS **Antergos 18.7**  system.

**Comment by xruth** [comment]
@akashnimare 
Yes I made the file executable first. Just to double check:
```
$ ./Zulip-2.3.5-x86_64.AppImage 
Segmentation fault (core dumped)
$ ls -l Zulip-2.3.5-x86_64.AppImage 
-rwxr-xr-x 1 user users 59400189  8. Aug 13:33 Zulip-2.3.5-x86_64.AppImage
```

@timabbott 
uname -a:
```
Linux 4.17.14-arch1-1-ARCH #1 SMP PREEMPT Thu Aug 9 11:56:50 UTC 2018 x86_64 GNU/Linux
```

**Comment by petergerten** [comment]
probably related to https://github.com/electron/electron/issues/13972

**Comment by xruth** [comment]
@petergerten 
Thanks for the hint. Problem seems to be with glibc. I fixed the problem on my client by installing glibc 2.28-4 from testing branch. Seems like they broke something with electron in v 2.28-1.
```
$ pacman -Ss glibc
core/glibc 2.28-1 (base) [installed: 2.28-4]
    GNU C Library
core/lib32-glibc 2.28-1 [installed: 2.28-4]
    GNU C Library (32-bit)
```
```
$ ./Zulip-2.3.5-x86_64.AppImage 
[13:45:15.246] [info] Checking for update
[13:45:17.218] [info] Update for version 2.3.5 is not available (latest version: 2.3.5, downgrade is disallowed).
```

**Comment by akashnimare** [comment]
Nice! Closing since it's an upstream issue.
